cardano-addresses.cabal

```diff
@@ -87,6 +87,7 @@ test-suite unit
   ghc-options: -Wall -Wcompat -fwarn-redundant-constraints -threaded -rtsopts -with-rtsopts=-N
   build-depends:
       QuickCheck
+    , base16-bytestring
```

Comment:
:thinking:  ? What's wrong with `memory` and `Data.ByteArray.Convert` ?

Reply:
indeed, B16.encode could be replaced by `encode EBase16 . T.encodeUtf8`
done in https://github.com/input-output-hk/cardano-addresses/pull/31/commits/bec1b0678ea08813980e8d243a76b95be8d478bb